transcendence-app/src/app.module.ts

```diff
@@ -4,7 +4,7 @@ import { AuthModule } from './auth/auth.module';
 import { validate } from './config/env.validation';
 import { APP_INTERCEPTOR } from '@nestjs/core';
 import { SocketModule } from './socket/socket.module';
-import { ChatRoomModule } from './chat-room/chat-room.module';
+import { ChatModule } from './chat/chat.module';
```

Comment:
Why do you change this name back to Chat instead of ChatModule?

Reply:
Because I thought we were only going to have a ChatModule to manage everything, connections, joining and leaving chatrooms, ... In slack there is a thread about this, structure of chat modules of the backend. I will name it accordingly